tests/zenko_tests/create_buckets.py

```diff
@@ -140,8 +140,8 @@ def create_azure_containers():
     credential = AzureNamedKeyCredential(name=AZURE_ACCOUNT_NAME,
             key=AZURE_SECRET_KEY)
     blob_service_client = BlobServiceClient(account_url=AZURE_BACKEND_ENDPOINT,
-            credential=credential)
-    
+            credential=credential,
+            connection_verify=False)
```

Comment:
```suggestion
            connection_verify=VERIFY_CERTIFICATES)
```

(and move definition of VERIFY_CERTIFICATES to a global function or variable, to reduce redundancy)